src/niamoto/core/plugins/transformers/class_objects/categories_extractor.py

```
@@ -45,37 +34,28 @@
+            # Pydantic validation handles structure and types (including List[str])
+            validated_config = self.config_model(**config)
+            return validated_config
+        except ValidationError as e:
+            # Re-raise Pydantic errors as DataTransformError for consistency
+            # Get plugin name from input config dict
+            plugin_name = config.get("plugin", "unknown_plugin")
             raise DataTransformError(
-                "axis.field must be specified", details={"config": config}
+                f"Invalid configuration for {plugin_name}: {e}",
+                details={"pydantic_error": e.errors(), "config": config},
             )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)